.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,23 @@
+on: push
```

Comment:
The name of the file is `pull_request.yml` but it runs on `push`.

I recommend using the generic `sultan-ci.yml` file name. 

```suggestion
on: push
```

Reply:
I originally intended to run on pull requests, I totally forget about changing this.